std/range/package.d

```diff
@@ -3877,7 +3894,6 @@ pure unittest
     assert(b == [ "c", "b", "a" ]);
 }
 
-///
```

Comment:
I don't think we should undocument this, I actually think it should come before the sort example.